src/components/gameScreen/gameScreen.container.js

```diff
@@ -33,15 +33,16 @@ export const GameScreenContainer = () => {
   const [isButtonPressed, setIsButtonPressed] = useState(false);
```

Comment:
```
  const questionIcon = require('../../assets/img/icons/questionIcon.png');
  const bookmarkIcon = require('../../assets/img/icons/bookmarkIcon.png');
```
don't write this kind of things inside of component

Reply:
please don't mix variables:
1. useRoute
2. useNavigation
3. states
4. refs
5. other variables